skorch/callbacks/scoring.py

```diff
@@ -81,6 +81,7 @@ def __init__(
     def _get_name(self):
         """Find name of scoring function."""
         if self.name is not None:
+            print("name", self.name)
```

Comment:
Could you remove this please? This is probably also the reason the tests fail.

Reply:
yea my bad